src/runner.js

```diff
@@ -0,0 +1,54 @@
+const { resolve } = require('path')
+const dpack = require('@etherpacks/dpack')
+const chalk = require('chalk')
+const ethers = require('ethers')
+const ganache = require("ganache")
+const { send } = require('minihat')
+module.exports = runner = {}
+
+runner.run = async (output_dir) => {
+    const options = { logging: { quiet: "true"}}
+    const provider = new ethers.providers.Web3Provider(ganache.provider(options))
+    const signer = provider.getSigner()
+    const multifab_pack = require('../lib/multifab/pack/multifab_full_hardhat.dpack.json')
+    const dapp = await dpack.load(multifab_pack, ethers, signer)
+    const multifab = await dapp._types.Multifab.deploy()
+    const src_output = require(resolve(`${output_dir}/SrcOutput.json`))
+    const tst_output = require(resolve(`${output_dir}/TestOutput.json`))
+    const src_contracts = Object.values(src_output.contracts).map((obj) => Object.entries(obj)[0])
```

Comment:
this is hard to understand because the lambda does not explain what it is grabbing from the object